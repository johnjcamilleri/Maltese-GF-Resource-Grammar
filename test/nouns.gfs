-- Treebank nouns test
-- Run using the following command:
-- ./_gf < test/nouns.gfs > test/nouns.out

se utf8
i AllMlt.gf
rf -lines -tree -file=test/nouns.trees | l -table
