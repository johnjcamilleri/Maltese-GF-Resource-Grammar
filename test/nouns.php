#!/usr/bin/php
<?php

// Timing
$time_start = microtime(true);

// Just some variables
$outfile = 'test/nouns.out';
$command = './_gf < test/nouns.gfs > '.$outfile;
$htmlfile = 'test/nouns.out.html';
$TEXT = '';

// ================

// Execute GF stuff and capture output
if (@$GLOBALS['argv'][1] == '--cached') {
	echo " Reading directly from {$outfile}\n";
} else {
	echo " Running GF\n";
	chdir( dirname(__FILE__) . '/../' );
	exec($command, $out, $return_status);
	if ($return_status != 0)
		die (" Failed.\n");
}
$output = file($outfile, FILE_IGNORE_NEW_LINES | FILE_SKIP_EMPTY_LINES);
if ($output === false)
		die (" Failed reading {$outfile}.\n");

// ================

// Recursive function for nesting as deep as we need
function nest($stack, &$branch) {
	if (count($stack) > 1) {
		$key = array_shift($stack);
		nest($stack, $branch[$key]);
	} else {
		$branch = $stack[0];
	}
}

// Process line by line & put into heirarchical array
$tree = array(); // overall structure
$ix = 0; // item index
$console_output = '';
foreach ($output as $n => $line) {

	// Skip timings
	if (preg_match('/^(> )?\d+ msec/', $line)) continue;

	// Process items
	elseif (preg_match('/^(> )?(\w) \. (.*)/', $line, $matches)) {

		// Have we reached a new entry?
		if ($matches[1]) {
			$ix++;
		}

		// Split at arrow and nest
		$columns = explode('=>',$matches[3]);
		$columns = array_map('trim', $columns);
		nest($columns, $tree[$ix][$matches[2]]);
	}

	// Anything else is compiler output
	else
		$console_output .= $line . "\n";
}

// ================

// Process recursively for output

function disp($branch, $depth=0) {
	if (is_array($branch)) {
		foreach ($branch as $k => $v) {
			//printf("\n%s%-15s", str_repeat("\t", $depth), $k);
			echo "<div><b>$k</b>";
			disp($v, $depth+1);
			echo "</div>";
		}
	} else {
		echo $branch == '[]' ? '' : "<em>$branch</em>";
	}
}
/*
function dispHTML($branch, $depth=0) {
	if (is_array($branch)) {
		foreach ($branch as $k => $v) {
			printf('<h%2$d>%1$s</h%2$d>'."\n", $k, $depth+1);
			dispHTML($v, $depth+1);
		}
	} else {
		echo "<p>$branch</p>\n";
	}
}
*/

ob_start();
foreach ($tree as $item) {
	disp($item);
	echo "<hr/>";
}
$TEXT_out = ob_get_contents();
ob_end_clean();


// ================

// Format into HTML
echo " Formatting to HTML\n";
$HTML = <<<HTML
<html>
<head>
	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
	<style type="text/css">
		body { font: 18px/1.5 serif; color:#999; }
		div { margin-left: 2em; }
		#text-output { margin:0; }
		#text-output b { display: inline-block; width: 10em; font-weight:normal; }
		#text-output em { font-style: italic; color:#000; }

		table { border-collapse:collapse; border-spacing:0; }
		tr:nth-child(odd) { background-color:#eee; }
		tr.separator { background-color:#000; font-size:2px; }
		td { padding:0.2em 1em; border-width:1px 0; border-style:solid; border-color:#ccc; }
		td:last-child { font-style:italic; }
	</style>
</head>
<body>


<div id="text-output">
{$TEXT_out}
</div>

</body>
</html>
HTML;


/*
<table>{$HTML_table}</table>
<a href="#compiler-output">Compiler output</a>
<pre id="compiler-output">{$HTML_console}</pre>
*/


// Save to file
echo " Writing to file $htmlfile\n";
@unlink($htmlfile);
file_put_contents($htmlfile, $HTML);

// ================

// Stop timing
$time_end = microtime(true);
$time_taken = $time_end - $time_start;
printf(" Done in %.2fs (completed at %s)\n", $time_taken, date('H:i'));
