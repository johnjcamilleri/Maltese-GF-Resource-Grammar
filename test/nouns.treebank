airplane_N
animal_N
apartment_N
apple_N
art_N
ashes_N
baby_N
back_N
bank_N
bark_N
beer_N
belly_N
bike_N
bird_N
blood_N
boat_N
bone_N
book_N
boot_N
boss_N
boy_N
bread_N
breast_N
brother_N2
butter_N
camera_N
cap_N
car_N
carpet_N
cat_N
ceiling_N
chair_N
cheese_N
child_N
church_N
city_N
cloud_N
coat_N
computer_N
country_N
cousin_N
cow_N
day_N
distance_N3
doctor_N
dog_N
door_N
dust_N
ear_N
earth_N
egg_N
enemy_N
eye_N
factory_N
fat_N
father_N2
feather_N
fingernail_N
fire_N
fish_N
floor_N
flower_N
fog_N
foot_N
forest_N
fridge_N
friend_N
fruit_N
garden_N
girl_N
glove_N
gold_N
grammar_N
grass_N
guts_N
hair_N
hand_N
harbour_N
hat_N
head_N
heart_N
hill_N
horn_N
horse_N
house_N
husband_N
ice_N
industry_N
iron_N
king_N
knee_N
lake_N
lamp_N
language_N
leaf_N
leather_N
leg_N
liver_N
louse_N
love_N
man_N
meat_N
milk_N
moon_N
mother_N2
mountain_N
mouth_N
music_N
name_N
neck_N
newspaper_N
night_N
nose_N
number_N
oil_N
paper_N
peace_N
pen_N
person_N
planet_N
plastic_N
policeman_N
priest_N
queen_N
question_N
radio_N
rain_N
reason_N
religion_N
restaurant_N
river_N
road_N
rock_N
roof_N
root_N
rope_N
rubber_N
rule_N
salt_N
sand_N
school_N
science_N
sea_N
seed_N
sheep_N
ship_N
shirt_N
shoe_N
shop_N
silver_N
sister_N
skin_N
sky_N
smoke_N
snake_N
snow_N
sock_N
song_N
star_N
steel_N
stick_N
stone_N
stove_N
student_N
sun_N
table_N
tail_N
teacher_N
television_N
tongue_N
tooth_N
train_N
tree_N
university_N
village_N
war_N
water_N
wife_N
wind_N
window_N
wine_N
wing_N
woman_N
wood_N
worm_N
year_N
