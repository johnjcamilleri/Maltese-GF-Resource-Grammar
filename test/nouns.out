=> Starting gf with custom -path flag...
                              
         *  *  *              
      *           *           
    *               *         
   *                          
   *                          
   *        * * * * * *       
   *        *         *       
    *       * * * *  *        
      *     *      *          
         *  *  *              
                              
This is GF version 3.2. 
License: see help -license.   
Bug reports: http://code.google.com/p/grammatical-framework/issues/list


Languages:
> 24 msec
> 4 msec
> + reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Common.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Cat.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Noun.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Verb.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Adjective.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Adverb.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Numeral.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Sentence.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Question.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Relative.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Conjunction.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Phrase.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Text.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Structural.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Idiom.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Tense.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/NumeralTransfer.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Transfer.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Grammar.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Lexicon.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/abstract/Lang.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/prelude/Predef.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/prelude/Prelude.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/common/ParamX.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/common/CommonX.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/PatternsMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/ResMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/CatMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/NounMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/VerbMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/AdjectiveMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/AdverbMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/MorphoMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/NumeralMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/SentenceMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/QuestionMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/RelativeMlt.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/prelude/Coordination.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/ConjunctionMlt.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/PhraseMlt.gfo
+ reading /home/john/Documents/MOLTO/GF (Darcs Repository)/lib/src/common/TextX.gfo
+ reading /home/john/Documents/MOLTO/Maltese RGL/maltese/OrthoMlt.gfo
- compiling /home/john/Documents/MOLTO/Maltese RGL/maltese/ParadigmsMlt.gf... checking module ParadigmsMlt
   Happened in overloading mkNoun in ParadigmsMlt.gf, line 0:
      no applicable case for N_Coll
OCCURRED IN
trying patterns N_Sg, N_Dl, N_Det
216 msec
> atomic term airplane_N
given _
   constant not found: airplane_N
4 msec
> atomic term animal_N
given _
   constant not found: animal_N
4 msec
> atomic term apartment_N
given _
   constant not found: apartment_N
8 msec
> atomic term apple_N
given _
   constant not found: apple_N
0 msec
> atomic term art_N
given _
   constant not found: art_N
4 msec
> atomic term ashes_N
given _
   constant not found: ashes_N
8 msec
> atomic term baby_N
given _
   constant not found: baby_N
4 msec
> atomic term back_N
given _
   constant not found: back_N
4 msec
> atomic term bank_N
given _
   constant not found: bank_N
4 msec
> atomic term bark_N
given _
   constant not found: bark_N
4 msec
> atomic term beer_N
given _
   constant not found: beer_N
4 msec
> atomic term belly_N
given _
   constant not found: belly_N
4 msec
> atomic term bike_N
given _
   constant not found: bike_N
4 msec
> atomic term bird_N
given _
   constant not found: bird_N
4 msec
> atomic term blood_N
given _
   constant not found: blood_N
4 msec
> atomic term boat_N
given _
   constant not found: boat_N
4 msec
> atomic term bone_N
given _
   constant not found: bone_N
8 msec
> atomic term book_N
given _
   constant not found: book_N
4 msec
> atomic term boot_N
given _
   constant not found: boot_N
4 msec
> atomic term boss_N
given _
   constant not found: boss_N
0 msec
> atomic term boy_N
given _
   constant not found: boy_N
4 msec
> atomic term bread_N
given _
   constant not found: bread_N
8 msec
> atomic term breast_N
given _
   constant not found: breast_N
4 msec
> atomic term brother_N2
given _
   constant not found: brother_N2
4 msec
> atomic term butter_N
given _
   constant not found: butter_N
8 msec
> atomic term camera_N
given _
   constant not found: camera_N
4 msec
> atomic term cap_N
given _
   constant not found: cap_N
4 msec
> atomic term car_N
given _
   constant not found: car_N
4 msec
> atomic term carpet_N
given _
   constant not found: carpet_N
0 msec
> atomic term cat_N
given _
   constant not found: cat_N
8 msec
> atomic term ceiling_N
given _
   constant not found: ceiling_N
4 msec
> atomic term chair_N
given _
   constant not found: chair_N
4 msec
> atomic term cheese_N
given _
   constant not found: cheese_N
4 msec
> atomic term child_N
given _
   constant not found: child_N
16 msec
> atomic term church_N
given _
   constant not found: church_N
4 msec
> atomic term city_N
given _
   constant not found: city_N
4 msec
> atomic term cloud_N
given _
   constant not found: cloud_N
4 msec
> atomic term coat_N
given _
   constant not found: coat_N
4 msec
> atomic term computer_N
given _
   constant not found: computer_N
4 msec
> atomic term country_N
given _
   constant not found: country_N
8 msec
> atomic term cousin_N
given _
   constant not found: cousin_N
0 msec
> atomic term cow_N
given _
   constant not found: cow_N
4 msec
> atomic term day_N
given _
   constant not found: day_N
4 msec
> atomic term distance_N3
given _
   constant not found: distance_N3
4 msec
> atomic term doctor_N
given _
   constant not found: doctor_N
8 msec
> atomic term dog_N
given _
   constant not found: dog_N
4 msec
> atomic term door_N
given _
   constant not found: door_N
4 msec
> atomic term dust_N
given _
   constant not found: dust_N
8 msec
> atomic term ear_N
given _
   constant not found: ear_N
4 msec
> atomic term earth_N
given _
   constant not found: earth_N
4 msec
> atomic term egg_N
given _
   constant not found: egg_N
4 msec
> atomic term enemy_N
given _
   constant not found: enemy_N
4 msec
> atomic term eye_N
given _
   constant not found: eye_N
4 msec
> atomic term factory_N
given _
   constant not found: factory_N
0 msec
> atomic term fat_N
given _
   constant not found: fat_N
8 msec
> atomic term father_N2
given _
   constant not found: father_N2
4 msec
> atomic term feather_N
given _
   constant not found: feather_N
4 msec
> atomic term fingernail_N
given _
   constant not found: fingernail_N
4 msec
> atomic term fire_N
given _
   constant not found: fire_N
4 msec
> atomic term fish_N
given _
   constant not found: fish_N
4 msec
> atomic term floor_N
given _
   constant not found: floor_N
4 msec
> atomic term flower_N
given _
   constant not found: flower_N
4 msec
> atomic term fog_N
given _
   constant not found: fog_N
8 msec
> atomic term foot_N
given _
   constant not found: foot_N
4 msec
> atomic term forest_N
given _
   constant not found: forest_N
4 msec
> atomic term fridge_N
given _
   constant not found: fridge_N
4 msec
> atomic term friend_N
given _
   constant not found: friend_N
8 msec
> atomic term fruit_N
given _
   constant not found: fruit_N
4 msec
> atomic term garden_N
given _
   constant not found: garden_N
8 msec
> atomic term girl_N
given _
   constant not found: girl_N
4 msec
> atomic term glove_N
given _
   constant not found: glove_N
4 msec
> atomic term gold_N
given _
   constant not found: gold_N
0 msec
> atomic term grammar_N
given _
   constant not found: grammar_N
8 msec
> atomic term grass_N
given _
   constant not found: grass_N
4 msec
> atomic term guts_N
given _
   constant not found: guts_N
4 msec
> atomic term hair_N
given _
   constant not found: hair_N
4 msec
> atomic term hand_N
given _
   constant not found: hand_N
0 msec
> atomic term harbour_N
given _
   constant not found: harbour_N
8 msec
> atomic term hat_N
given _
   constant not found: hat_N
4 msec
> atomic term head_N
given _
   constant not found: head_N
8 msec
> atomic term heart_N
given _
   constant not found: heart_N
4 msec
> atomic term hill_N
given _
   constant not found: hill_N
4 msec
> atomic term horn_N
given _
   constant not found: horn_N
4 msec
> atomic term horse_N
given _
   constant not found: horse_N
4 msec
> atomic term house_N
given _
   constant not found: house_N
4 msec
> atomic term husband_N
given _
   constant not found: husband_N
4 msec
> atomic term ice_N
given _
   constant not found: ice_N
4 msec
> atomic term industry_N
given _
   constant not found: industry_N
4 msec
> atomic term iron_N
given _
   constant not found: iron_N
4 msec
> atomic term king_N
given _
   constant not found: king_N
4 msec
> atomic term knee_N
given _
   constant not found: knee_N
4 msec
> atomic term lake_N
given _
   constant not found: lake_N
8 msec
> atomic term lamp_N
given _
   constant not found: lamp_N
4 msec
> atomic term language_N
given _
   constant not found: language_N
4 msec
> atomic term leaf_N
given _
   constant not found: leaf_N
8 msec
> atomic term leather_N
given _
   constant not found: leather_N
0 msec
> atomic term leg_N
given _
   constant not found: leg_N
4 msec
> atomic term liver_N
given _
   constant not found: liver_N
8 msec
> atomic term louse_N
given _
   constant not found: louse_N
4 msec
> atomic term love_N
given _
   constant not found: love_N
4 msec
> atomic term man_N
given _
   constant not found: man_N
4 msec
> atomic term meat_N
given _
   constant not found: meat_N
8 msec
> atomic term milk_N
given _
   constant not found: milk_N
4 msec
> atomic term moon_N
given _
   constant not found: moon_N
4 msec
> atomic term mother_N2
given _
   constant not found: mother_N2
8 msec
> atomic term mountain_N
given _
   constant not found: mountain_N
4 msec
> atomic term mouth_N
given _
   constant not found: mouth_N
8 msec
> atomic term music_N
given _
   constant not found: music_N
0 msec
> atomic term name_N
given _
   constant not found: name_N
8 msec
> atomic term neck_N
given _
   constant not found: neck_N
4 msec
> atomic term newspaper_N
given _
   constant not found: newspaper_N
4 msec
> atomic term night_N
given _
   constant not found: night_N
4 msec
> atomic term nose_N
given _
   constant not found: nose_N
8 msec
> atomic term number_N
given _
   constant not found: number_N
4 msec
> atomic term oil_N
given _
   constant not found: oil_N
4 msec
> atomic term paper_N
given _
   constant not found: paper_N
4 msec
> atomic term peace_N
given _
   constant not found: peace_N
4 msec
> atomic term pen_N
given _
   constant not found: pen_N
8 msec
> atomic term person_N
given _
   constant not found: person_N
4 msec
> atomic term planet_N
given _
   constant not found: planet_N
4 msec
> atomic term plastic_N
given _
   constant not found: plastic_N
4 msec
> atomic term policeman_N
given _
   constant not found: policeman_N
4 msec
> atomic term priest_N
given _
   constant not found: priest_N
4 msec
> atomic term queen_N
given _
   constant not found: queen_N
4 msec
> atomic term question_N
given _
   constant not found: question_N
8 msec
> atomic term radio_N
given _
   constant not found: radio_N
4 msec
> atomic term rain_N
given _
   constant not found: rain_N
4 msec
> atomic term reason_N
given _
   constant not found: reason_N
8 msec
> atomic term religion_N
given _
   constant not found: religion_N
4 msec
> atomic term restaurant_N
given _
   constant not found: restaurant_N
4 msec
> atomic term river_N
given _
   constant not found: river_N
4 msec
> atomic term road_N
given _
   constant not found: road_N
4 msec
> atomic term rock_N
given _
   constant not found: rock_N
4 msec
> atomic term roof_N
given _
   constant not found: roof_N
8 msec
> atomic term root_N
given _
   constant not found: root_N
4 msec
> atomic term rope_N
given _
   constant not found: rope_N
4 msec
> atomic term rubber_N
given _
   constant not found: rubber_N
8 msec
> atomic term rule_N
given _
   constant not found: rule_N
4 msec
> atomic term salt_N
given _
   constant not found: salt_N
8 msec
> atomic term sand_N
given _
   constant not found: sand_N
0 msec
> atomic term school_N
given _
   constant not found: school_N
8 msec
> atomic term science_N
given _
   constant not found: science_N
4 msec
> atomic term sea_N
given _
   constant not found: sea_N
4 msec
> atomic term seed_N
given _
   constant not found: seed_N
8 msec
> atomic term sheep_N
given _
   constant not found: sheep_N
4 msec
> atomic term ship_N
given _
   constant not found: ship_N
4 msec
> atomic term shirt_N
given _
   constant not found: shirt_N
4 msec
> atomic term shoe_N
given _
   constant not found: shoe_N
8 msec
> atomic term shop_N
given _
   constant not found: shop_N
4 msec
> atomic term silver_N
given _
   constant not found: silver_N
4 msec
> atomic term sister_N
given _
   constant not found: sister_N
8 msec
> atomic term skin_N
given _
   constant not found: skin_N
4 msec
> atomic term sky_N
given _
   constant not found: sky_N
4 msec
> atomic term smoke_N
given _
   constant not found: smoke_N
8 msec
> atomic term snake_N
given _
   constant not found: snake_N
4 msec
> atomic term snow_N
given _
   constant not found: snow_N
4 msec
> atomic term sock_N
given _
   constant not found: sock_N
4 msec
> atomic term song_N
given _
   constant not found: song_N
8 msec
> atomic term star_N
given _
   constant not found: star_N
4 msec
> atomic term steel_N
given _
   constant not found: steel_N
4 msec
> atomic term stick_N
given _
   constant not found: stick_N
8 msec
> atomic term stone_N
given _
   constant not found: stone_N
4 msec
> atomic term stove_N
given _
   constant not found: stove_N
4 msec
> atomic term student_N
given _
   constant not found: student_N
4 msec
> atomic term sun_N
given _
   constant not found: sun_N
4 msec
> atomic term table_N
given _
   constant not found: table_N
8 msec
> atomic term tail_N
given _
   constant not found: tail_N
4 msec
> atomic term teacher_N
given _
   constant not found: teacher_N
12 msec
> atomic term television_N
given _
   constant not found: television_N
4 msec
> atomic term tongue_N
given _
   constant not found: tongue_N
4 msec
> atomic term tooth_N
given _
   constant not found: tooth_N
8 msec
> atomic term train_N
given _
   constant not found: train_N
4 msec
> atomic term tree_N
given _
   constant not found: tree_N
4 msec
> atomic term university_N
given _
   constant not found: university_N
8 msec
> atomic term village_N
given _
   constant not found: village_N
4 msec
> atomic term war_N
given _
   constant not found: war_N
4 msec
> atomic term water_N
given _
   constant not found: water_N
8 msec
> atomic term wife_N
given _
   constant not found: wife_N
4 msec
> atomic term wind_N
given _
   constant not found: wind_N
4 msec
> atomic term window_N
given _
   constant not found: window_N
4 msec
> atomic term wine_N
given _
   constant not found: wine_N
4 msec
> atomic term wing_N
given _
   constant not found: wing_N
4 msec
> atomic term woman_N
given _
   constant not found: woman_N
4 msec
> atomic term wood_N
given _
   constant not found: wood_N
4 msec
> atomic term worm_N
given _
   constant not found: worm_N
8 msec
> atomic term year_N
given _
   constant not found: year_N
4 msec
> See you.
